### PR TITLE
[Sage-196] Input Helper - Deprecation

### LINKED ISSUE #196: Deprecate Input Helper component
## Description
The [Input Helper component](https://sage-design-system.kajabi.com/pages/component/input_helper) was originally intended for work in the Offer checkout and is no longer relevant.

We may also want to consider removing or repurposing the related [Input Group](https://sage-design-system.kajabi.com/pages/component/input_group) component as well.

### PR DESCRIPTION
**Pull request by anechol**
## Description

Deprecates the Input Helper Sage component.

## Screenshots

N/A

## Testing in `sage-lib`

Check that there no references to the Input Helper component left in the Sage code.

## Testing in `kajabi-products`

(**LOW**) Deprecates the Input Helper Sage component. There are no usages in Kajabi Products, but a small patch will go out to remove one instance of a deprecated Input Group prop that previously referenced Input Helper.

## Related

Closes #196

### CODE REVIEW

File: packages/sage-system/lib/init.js [hunk 1]
```diff
@@ -102,11 +102,6 @@ Sage.init = function(elementNamesToInitLegacy) {
     Sage.inputgroup.init();
   }
 
-  // Initialize Input groups
-  if ( shouldInitLegacy('inputhelper', '.sage-input-helper') ) {
-    Sage.inputhelper.init();
```
Comment (monkeypox8) [inline packages/sage-system/lib/init.js:107]: Looking great! A few more files for cleanup associated with this line:

- `/docs/lib/sage-frontend/javascript/docs/inputhelper.js`
- `/packages/sage-system/lib/inputhelper.js`
- `zxcvbn` in `packages/sage-system/package.json` (pw strenght lib loaded by the JS file above)

Comment (anechol) [reply packages/sage-system/lib/init.js:107]: Oof nice catch!

**Review by monkeypox8 — APPROVED** [review]
👍🏼 LGTM! Great work!

File: packages/sage-system/package.json [hunk 2]
```diff
@@ -54,7 +54,6 @@
     "arrive": "^2.4.1",
     "babel-loader": "^8.1.0",
     "sortablejs": "^1.12.0",
-    "tiny-slider": "^2.9.4",
-    "zxcvbn": "^4.4.2"
```
Comment (QuintonJason) [inline packages/sage-system/package.json:58]: 🔬 💥

**Review by QuintonJason — APPROVED** [review]

**Pull request merged**